CMake/Dependencies/libkvsCommonLws-CMakeLists.txt

```diff
@@ -6,7 +6,7 @@ include(ExternalProject)
 
 ExternalProject_Add(libkvsCommonLws-download
     GIT_REPOSITORY    https://github.com/awslabs/amazon-kinesis-video-streams-producer-c.git
-    GIT_TAG           3f437b936adf1899533a8f82c9f374e7041004eb
+    GIT_TAG           15f124e5a5561679cf001bb36c2c758073740681
```

Comment:
anything in kvsCommonLws could affect the bug?

Reply:
We implemented blocking shutdown in the timer queue in PIC which was pushed into common and now in here. This is needed for proper termination sequence for the PeerConnection. None directly involved in the presumed fix for the crash you are seeing